mentat/code_feature.py

```diff
@@ -1,42 +1,83 @@
+from __future__ import annotations
+
 import asyncio
 import math
 import os
 from enum import Enum
 from pathlib import Path
+from typing import Optional
 
-from mentat.code_map import get_code_map
+from mentat.code_map import get_code_map, get_ctags
 from mentat.diff_context import annotate_file_message, parse_diff
 from mentat.git_handler import get_diff_for_file
+from mentat.interval import Interval, parse_intervals
 from mentat.llm_api import count_tokens
 from mentat.session_context import SESSION_CONTEXT
 from mentat.utils import sha256
 
+MIN_INTERVAL_LINES = 10
```

Comment:
Any reason why "10" (arbitrary number or works best for a specific reason)? Just curious